package.json

```diff
@@ -261,6 +261,18 @@
           }
         }
       }
+    ],
+    "languages": [
+      {
+        "id": "blend",
+        "extensions": [".blend"],
```

Comment:
Oh, btw, does automatic save in blender still adds number at the end of file creating new file extensions?
Like .blend1, .blend2... I think it used to do that back in a day

Reply:
AFAIK this still works, but IMO displaying icons for all the `.blend*` backup files will clutter the UI. I think we should just display the Blender icon for main `.blend` files and keep the backups 'grayed out'.